web/src/components/pages/profile/info/info.tsx

```diff
@@ -323,25 +323,52 @@ function ProfilePage({
             </Tooltip>
 
             <div className="checkboxes">
+              {/* TODO(riley): Once Open Voice Challenge backend is set up,
+                               only show this to enrolled users. */}
+              {/* TODO(riley): Connect `onChange` and `checked` to the user
+                               model. */}
+              {false && (
+                <LabeledCheckbox
+                  label={
+                    <>
+                      <strong>Get updates about the challenge</strong>
+                      Receive Open Voice Challenge emails that include weekly
+                      recaps and announcements.
+                    </>
+                  }
+                  onChange={() => {}}
+                  checked={false}
+                />
+              )}
               <LabeledCheckbox
                 label={
-                  <Localized id="email-opt-in-info">
-                    <span />
-                  </Localized>
+                  <>
+                    <Localized id="email-opt-in-info-title">
+                      <strong />
+                    </Localized>
+                    <Localized id="email-opt-in-info-sub">
+                      <span />
+                    </Localized>
+                  </>
                 }
                 onChange={handleChangeFor('sendEmails')}
                 checked={sendEmails}
               />
 
               {!user.account && !isSubmitted && (
-                <React.Fragment>
+                <>
```

Comment:
Any reason to use `React.Fragment` over `<>`?

Reply:
I think the TS compiler needed a little longer to be ready for it, so no good reason not to use `<>` now :)